packages/nodes-base/credentials/BeeminderApi.credentials.ts

```diff
@@ -13,12 +13,6 @@ export class BeeminderApi implements ICredentialType {
 	documentationUrl = 'beeminder';
 
 	properties: INodeProperties[] = [
-		{
```

Comment:
should be backwards compatible, user is not required anymore, defaulting to "me". tested